mlte/spec/__init__.py

```diff
@@ -1,6 +0,0 @@
-from .requirement import Requirement
```

Comment:
It looks like you caught on that I am trying to get rid of some of these re-exports from sub-packages. I know it makes things slightly more verbose, but it has solved so many import issues across the package that I think it is worth it. Thanks!

Reply:
Yes, trying to follow that convention. It is a bit more verbose, but actually I'm used to doing it that way, and getting rid of the circular import issues makes it really worth it.